niceml/mkdocs/mdjob.py

```diff
@@ -1,7 +1,7 @@
 """Module for generating mkdocs str for jobs"""
 from typing import List
 
-from dagster.core.definitions import NodeDefinition
+from dagster._core.definitions import NodeDefinition
```

Comment:
This is a `private` module and should not be used outside of dagster itself. 
Is there another / a better class that could be used for this purpose?